app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_store = {}
+
+    def wrapper(*args: tuple) -> Callable:
```

Comment:
The return type annotation for the `wrapper` function should be the same as the return type of the `func` it wraps, not `Callable`. Consider changing it to match the return type of `func`.